app/views/guests/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<h2><%= @guest.name %></h2>
+
+<h3>List of Rooms</h3>
+  <% @guest.rooms.each do |room| %>
+    <p>Suite: <%= room.suite %><br>
+    <p>Nightly Rate: $<%= room.rate %><br>
+    <p>Hotel: <%= room.hotel.name %><br>
+  <% end %>
+
+<h4>Add a room to <%= @guest.name %></h4>
+  <%= form_with url: "/guests/#{@guest.id}", method: :post do |form|%>
```

Comment:
As mentioned above, a more RESTful option here would be to direct this to a joins controller (GuestRoomsController) rather than back to the Guest controller, as this is going to create a join record between an existing room and guest record. A good path for this could be "/guests/:guest_id/rooms"